### PR TITLE
Add Best Buy

### PR DESCRIPTION
**Pull request by Rfosu2k2**
Add Best Buy for not having any 2FA or MFA

### CODE REVIEW

**Review by Rfosu2k2 — COMMENTED** [review]
This is the correct update with img/data

**Review by kmpoppe — DISMISSED** [review]
Alexa rank 192

File: _data/retail.yml [hunk 1]
```diff
@@ -55,6 +55,13 @@ websites:
       facebook: Arcaplanet
       lang: it
 
+    - name: Best Buy
+      url: https://www.bestbuy.com
+      img: bestbuy.png
+      tfa: No
+      twitter: BestBuy
```
Comment (Carlgo11) [inline _data/retail.yml]:
```suggestion
      twitter: BestBuySupport
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Try this image instead:
bestbuy_logo

File: _data/retail.yml (hunk 1 above)
Comment (Rfosu2k2) [reply _data/retail.yml]: bestbuy


How about this with the blue background?

**Comment by Rfosu2k2** [comment]
Should we use this image with the blue background?
bestbuy

**Review by Rfosu2k2 — COMMENTED** [review]
Updated per suggestion

**Comment by Carlgo11** [comment]
Either one works.

**Comment by Carlgo11** [comment]
https://github.com/2factorauth/twofactorauth/pull/4043#pullrequestreview-294607489 @rfosu2k2 you didn't actually update anything. Did you forget to upload the changes?

**Comment by Carlgo11** [comment]
Closing as the fork is no longer available.

**Pull request closed without merging**